src/depackers/xz.h

```diff
@@ -241,7 +241,27 @@ XZ_EXTERN void xz_dec_end(struct xz_dec *s);
  * care about the functions below.
  */
 #ifndef XZ_INTERNAL_CRC32
+#	ifdef __KERNEL__
```

Comment:
Do we need to add these back?

Reply:
No, but it reduces the (sizeable) diff between libxmp and XZ Embedded a little bit.

Reply:
Note the vast majority of the diff was changing stdint.h types to libxmp's custom fixed-width types, so this isn't even that big, and I can just remove these.

Reply:
I'd suggest keeping the changes to minimum

Reply:
OK, I ditched the `__KERNEL__` parts of these.